rules/use-figma-dev-mode/rule.md

```diff
@@ -0,0 +1,47 @@
+seoDescription: "Learn how Figma Dev Mode streamlines developer handovers by
+  providing a seamless bridge between design and development. type: rule"
+uri: use-figma-dev-mode
+authors:
+  - title: Betty Bondoc
+    url: https://www.ssw.com.au/people/betty-bondoc/
+created: 2024-10-30T14:20:35.753Z
+guid: 83be089e-9cad-4d55-8f77-26b1ce2d2fbc
+---
+
+One of the biggest pain points during design-to-development handover is the loss of information or miscommunication between teams. Designers might create stunning designs, but without clear specifications, developers can struggle to implement them accurately. This often results in back-and-forth clarifications, missed deadlines, and frustration.
+
+<!--endintro-->
+
+Figma Dev Mode solves this issue by offering developers all the necessary design details in a developer-friendly format. With access to measurements, code snippets, and assets, developers can easily translate designs into code without needing constant input from designers.
+
+## Benefits of Using Figma Dev Mode
+
+- **Reduced Friction:** All design assets are presented in a format developers can immediately use, reducing misunderstandings.
+- **Code Snippets Ready to Use:** Extract CSS, Swift, or Android code directly from the design elements.
+- **Access to Design Properties:** Measurements, spacing, and font details are available in one place, eliminating the need to switch between tools.
+- **Downloadable Assets:** Developers can directly download production-ready images and icons from Figma without requesting files from designers.
+- **Version Control Visibility:** See updates and changes in real-time, ensuring everyone is aligned with the latest design iteration.
+
+## How to Use Dev Mode Effectively
+
+1. **Enable Dev Mode:** Designers need to switch to Dev Mode before handing over a file.
+2. **Use Comments for Clarifications:** Encourage designers to leave notes for developers on any specific behaviors or constraints.
+3. **Organize Components:** Properly named layers and components help developers quickly find relevant assets.
+4. **Test Asset Downloads:** Ensure assets are correctly prepared and named for development to prevent rework.
+
+### Good Practices for Dev Mode Handovers
+
+::: bad  
+![Figure: Bad Example - A basic mockup or prototype with no technical details, forcing developers to guess measurements and styles.](figma-preview.png)  
+:::
+
+::: good  
+![Figure: Good Example - The design includes useful developer details such as color codes, font styles, measurements, and spacing.](figma-dev-mode.png)  
+:::
+
+### Watch this video on Figma Dev Mode
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/__ABPkb0aF8?si=gV7URLi-X5NiPs4y" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
YouTube videos can be embedded much simpler. Have a look at: https://www.ssw.com.au/rules/rule/#8-videos